ush/parsing_namelists_FV3.sh

```diff
@@ -251,7 +251,7 @@ EOF
   cat >> input.nml << EOF
   iovr         = ${iovr:-"3"}
   ltaerosol    = ${ltaerosol:-".false."}
-  lradar       = ${lradar:-".false."}
+  lradar       = ${lradar:-".true."}
```

Comment:
this is no longer optional with the update in `diag_table`, correct?
In other words, if `lradar=.false.` and `diag_table` has `refl_10cm`, will the model still run successfully?

Reply:
lradar=.true. is needed to have correct refl_10cm  values. Can this be made optional? When lradar = .true., refl_10cm will be added to the diag_table?

Reply:
When I accidentally left this as false during testing, the model still completed successfully (and the fields were missing from the output). So both settings are valid.

Reply:
Interesting. That means, if `lradar=.false.`, the `refl_10cm` values are incorrect even if the model runs successfully.